constants.js

```diff
@@ -48,17 +49,21 @@ const checksWhitelist = {
       wipCheck,
       jobCheck,
       modelCheck,
-      prTemplateCheck
+      prTemplateCheck,
+      codeOwnerCheck
     ],
     [reopenEvent]: [
       changelogCheck,
       branchCheck,
       wipCheck,
       jobCheck,
       modelCheck,
-      prTemplateCheck
+      prTemplateCheck,
+      codeOwnerCheck
     ],
     [PRLabelEvent]: [assigneeCheck, prLabelCheck],
+    [synchronizeEvent]: [mergeConflictCheck, jobCheck, modelCheck, codeOwnerCheck],
+    [closeEvent]: [allMergeConflictCheck],
     [synchronizeEvent]: [mergeConflictCheck, jobCheck, modelCheck],
     [closeEvent]: [allMergeConflictCheck, updateWithDevelopCheck],
     [editEvent]: [wipCheck],
```

Comment:
Should we have this for edit event as well - is the event triggered on files in PR being changed as well?

Reply:
The edit event is for things like description getting edited or body of the comment getting edited.

Reply:
Ah ok!